TheWarWithin/ShamanEnhancement.lua

```diff
@@ -2741,4 +2742,4 @@ spec:RegisterSetting( "filler_shock", true, {
3w9ZndGOjUj7O85FKq5Hul))fZOpL9pv1Vc1K(D3sS0Vml)UBVnE1FsVXbzlItexJTIHcNeNn7CkWjDlT3(vOjzE)SHNx1nZgEq97Kn8SKV80FoEXmEri1Uo29G9abVI1bX(z6TDRU(2nD)8ZSRe3mxELE9vRUt0ggXQH2SG023bCWddhWme3S9Ggr1g2joGAXQn4asgfvyYFJSH1gILX9fTMmn(mWBX8Py3G5bNs(K(W3z5tnUVY3m7Qrdof9wk3kq38o4wb7vHGdEzKpTbX0ViTNIFjAPFf7BmV)uNwf9UwIeNco9kkf(9ND3qCXTQvfXHVmX7lMRLDYF0FYzE1QiiNw1boqrbssqnXd57NREhFND5OrYtG)w2snyFCRHrxFYe3(1)X27bFHDLXo6hXFJDvWz691r(pqBZuCz0Qkbay7NFw6nrC2y8phW(T6NYi2hC38WPPhRtEO4OhcIR0NIqHtzMd)wCqL60ZP0NA2flDxx7e81UQLzzL7obCheDpqIoWzAOtpsdFQzxSq09oeI(B4bAaxRgkCPUqTg(yoGdx6Xc(watE4W0ozv4eyZcoxyNs(Nal6B4yK1k5oW3xLj2piTcxHSj6kAhs2(y0vxzYMOlFDhv7JqlkLjzQDMQ7OBiDa7N5USrIn(FuxDjkPK(K28nsogFNeTCkzPOuhU3J1zphL7hNUyd96C1bezMVcVrTJAn2Rdtxqk8F10gPy9rRvF7CZgvSsiHvuOQcFqVoLszby8wjy22l5Ds55hvrRtqZPaWBy772HMlkdV6p3LCXA(785NPz3ewYJcsh8CVibz2wHBg3JiOk)CC2c4QQCcwxv13x0MK5hYrQQKA5jUjjKG)8v1JPNcFRtVzupto5jUK7aGjaDQtWKEQwE4xKaqTohkDUuTAatznmGH1Uj0uSGr8QF3Qy0SRv(tIliVvY8SPMv(wJO2tXMgxS4MzE9GkJkXK6P0FxAmXGkWsa5WWdYycCHOjAI3ZqW7(MVEx3mzuL7b4xTRzEdCqUGyEyCN9dh3tgYL96xDB19xKZbXGeIRRHxvJXYUoHqJow9fyhfVAicsMAYGRJO)qJl5rOcBfKiJh5givRgnDM2VM2o8RedW6OInhktMnUzzLRHk9hKU9vH6MzF5Gfy3FJeximz7gF2vy6lWB50QYifh(IF08hLctu93DqIkqpRDe6Mzt6oOCSouYM(zaWz16mMaTWA37dwykkdZMSBlSVdz8F1LqZP1rjwSzDuE1Jhx1yNvpujXOtijILWnR2KOyaAWGQ6cb82h2gC2HipG9iGOanslPMW3uS7QAzx3(ownT3fixQG4CdyU0f5WKwcq15Gy9n5PjWD2MYKXvfg4)DQn7OCQseKJeXBmJGkVmR0FjVtIDOjbIzQyIP(YmFP3ULRNv9atUdB8okRCfQ1CGqgbNALSrVQsIc)mBsSJLvu0tjr2Rj5ME4HKqrH(WH596RoL3gSluwXp8QYIY7zqgZM2Ua(pwbUFeOZtXCuwIwAb8W62rFixaRSAiR3hXWETOYazReOwNDfh9ayLgk13cXO1cAnw)qKaQt5xYmzn33D0ohbauXijI73(6gpBO8ijP6eYpPv(b6PNISWCZeqHinxuoHIdXdfkT)448s6rLQX9L0dIyVkLoPhHXAQEXZX5SFDQfs9oelWLJEdm14qc4VFMDaGsT3bpjzBGcBcqU7b)4VoB0oKNA7DUQMaFnXIL4mHALh9a9cb0i7rJvI8Tej5YCsEcdNkF6n0lpcWxfuW)k32dEpbfi7Sjewdr0Nboss8vEeeFXWMDaD57u)mTeDA(Ee3biMDS7dzh9qu)GOBcYn2k3fyg3y67dvP947BPhK2BhPLuTwxomvU6jM40Eqz2jlWMVjSTq(WkIy1KQDRCqJkxCtlggeZtyxayXAB1(GYicTApgpTMUAvWU46uIx7MTc991prSuaM7n14zLfiij2X6xO67w9a(XFUneX9Nw1S8bzYmxLxyvEx4SSjUkXCDGA4G4(1GKb6GVuyhayqTDR9lf)ra1CtF)TyIa(CktBtmFcq)i1O3y3zJPn)34po1GoOjIaql6JelhnyhdwX1KnUNfwbzCs0S97dRBO1)wbcOpTlrciU2n1(j75Qqwg1WYSGlIbz1l1Gwd46cRVTvdJ8cTlsO049DFaMp)Ei08RvP5d6HmTl1Xg9Awbi64UEXvkdQ6kO2u0)fBP1fAJVn6jJAtyhn82My8TuPGQ0qNjvsxI8dyTkhmWA3BBHqRD7g6wbOvdzFP)7hiQVBWhBa(ynjC0cdMS9vREmCroZx)ERNUNDjjydINz)GHlrKT((wryCh9WSljic8q0QK0kZQwSlH7k0vi27SfoNPMWxrW8WmJb0AO8vux144ThojCa(AmCJDqnkn74VzkoMlcKbkPFjqE9vWabhi8Ey4DqimI4xzaQxcLFTpAVtIwisR0T3kkSALyquPfOPRIVY)EJmVKmEz5NQdtJL7CKB8Ta63KdJW2UapovlejCLgufuhy1EXh9(i4WLLJX7UpvGOxCwIeqZCNGzNtoJo(XJ8m5dSkWGivQDpZrAWbQxcaGTREdUcx5nifubed0qbyLRFSZmcUf)ospcGCS7DlYPLb42j1kwr1AdCoFomAP4O7BEPWVXCmX9MgIGPBKrsV8id21QR7DkyneA5yeD0z4UkxwLR87sYgtSCfK8IERx1gtIqPncu0cce(OKOSnDKU0hkdW0USkhykRaifCkHIHHYkumiyWIhpNkQh(NkxdMTmM3l5NYsczxHuc8eL(y5sc1IF4Pirse(nwxs4PaYLwoYKXDQEVUw)U2q1baAqBWpkLhOzLq71tpBYOx7sgfI8EmQbrJYO7y93RYqCZm1Kib26Jr2(n4XHNWVjbgAFoS0(bxNFMZ8FxeKus(3UjDZKNlBt6)TZwH9892Zm4bYCYoYAvJWwOuUrY9L949tziiwI889Wvoc4SlERz5D(U5qggbCwrcmtXOfLG97H5D)N]] )
```

Comment:
Was there a revision to the Enhancement priority?  If so, the text version needs to be updated.

Reply:
Ticket https://github.com/Hekili/hekili/issues/3867 claims that the .simc file was updated in the addon, but the pack string is the same as it was before those changes